common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1078,6 +1078,11 @@
          */
         public static final String BROWSER_EXT_WEB_CP = "companyportal://";
 
+        /**
+         * Prefix in the redirect from WebCP.
```

Comment:
>Prefix in the redirect from WebCP. [](start = 11, length = 34)

This comment matches the above shouldn't it mention the authenticator

Reply:
oh yes, thanks for pointing it out, will make the change.